tests/func/test_get.py

```diff
@@ -87,3 +89,19 @@ def test_get_to_dir(dname, erepo):
 
     assert os.path.isdir(dname)
     assert filecmp.cmp(erepo.FOO, dst, shallow=False)
+
+
+def test_get_from_non_dvc_master(empty_dir, erepo, caplog):
+    shutil.rmtree(erepo.dvc.dvc_dir)
+    erepo.dvc.scm.repo.index.remove([erepo.dvc.dvc_dir], r=True)
+    erepo.dvc.scm.commit("remove .dvc")
+
+    caplog.clear()
+    imported_file = "foo_imported"
+    with caplog.at_level(logging.INFO, logger="dvc"):
+        Repo.get(erepo._root_dir, erepo.FOO, out=imported_file, rev="branch")
+
+    assert caplog.text == ""
```

Comment:
Hm, why do we test that there is no info output to logger?

Reply:
https://github.com/iterative/dvc/issues/2858 
I think that we should use `caplog` more often in order to prevent spawning of UI issues that are quite common recently. Do you think I should also check `capsys`?

Reply:
You'll probably get progress bars in capsys, not easy to test, especially automatically, but your call, I don't have a strong opinion here 🙂

Reply:
pbars go to stderr by default, so shouldn't be an issue.

Reply:
I wonder why `Repo.get()` has not output though.